### ISSUE TITLE
Wrong number formatting; adds '0' before.

### ISSUE THREAD

**Issue by lukasborawski** [EducationLink/vue-tel-input#126]
I have predefined input number: `791315354` and it's Poland `PL` number. After formatting, I get `0` before the number and with international mode, I've `+93 79 131 5354` which is Afganistan number 😃 . My settings:

```
  inputTelProps: {
    mode: 'international',
    placeholder: 'Phone Number *',
    validCharactersOnly: true,
    inputOptions: {
      showDialCode: true,
      tabindex: ''
    }
  },
```

However the number `889121221` is formatted correctly and I get `+48 889 121 221`.

**Comment by iamstevendao** [comment]
@lukasborawski, sorry for the delay. It should be fixed in [v3.1.6](https://github.com/EducationLink/vue-tel-input/releases/tag/v3.1.6), feel free to re-open if you still face the same issue.

**Comment by 5hanth** [comment]
@iamstevendao not sure what was the fix pushed in v3.1.6 but I'm facing this in 4.3.0
